docs/user/attestation-provider.md

```diff
@@ -0,0 +1,113 @@
+# Becoming an Attestation Provider
+
+This guide is a walk through to become an [attestation provider](#attestation-provider).
+There are two situations in which it makes sense to do so:
+
+* Running local attestation providers, without any capital requirement, to act as a safeguard against malicious forks ;
+* Running an incentivized attestation provider to reap rewards, which contributes to the network and helps improve the decentralization of the network and provides attestations for the [state connector](#state-connector).
```

Comment:
"network" repeated.

Reply:
Fixed in [`48bd9e9` (#14)](https://github.com/flare-foundation/docs/pull/14/commits/48bd9e961a8f1c0053cc49e21e9f6a25bf738195)